ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+WKNavigationDelegate.swift

```diff
@@ -599,6 +599,26 @@ extension BrowserViewController: WKNavigationDelegate {
     let response = navigationResponse.response
     let responseURL = response.url
     let tab = tab(for: webView)
+    let isInvalid: Bool
+    if let httpResponse = response as? HTTPURLResponse {
+      isInvalid = httpResponse.statusCode >= 400
```

Comment:
Are you sure about this >= 400 check? What happens if the response is a 000?

Reply:
I think so: 
https://github.com/brave/brave-core/blob/c4daecc6b66e088f1eee0a3c6bfe04bd0bbba9a5/chromium_src/chrome/browser/ssl/https_upgrades_interceptor.cc#L44